src/formpack/pack.py

```diff
@@ -176,6 +179,9 @@ def load_version(self, schema):
 
         self.versions[form_version.id] = form_version
 
+    def extend_survey(self, analysis_form: Dict) -> None:
```

Comment:
I used to use `: dict` . I never thought about using `typing import Dict` .

Reply:
Interesting... which do you prefer? With `Dict` we can also be more specific like `Dict[str, str]`

Reply:
We are now imported `from __future__ import annotations` 

`def extend_survey(self, analysis_form: dict) -> None:`

and `dict[str, str]` should work too.

Reply:
now tracked by #306